doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/transform_cli.html

```diff
@@ -349,6 +351,26 @@ <h4>Other options</h4>
 		<dd>Provide this flag if you trust any Groovy transformation
 			functions used in the provided project. This will lift restrictions
 			on Groovy calls and speed up the execution.</dd>
+		<dt><code>-statisticsOut &lt;statistics-file&gt;</code></dt>
+		<dd>
+			Specifies the location of a file to write the transformation statistics to.
+			The transformation statistics are saved as JSON.
+			They contain aggregated information on the different tasks that were run as part of the transformation.
+		</dd>
+		<dt><code>-successEvaluation &lt;file-or-URI-to-script&gt;</code></dt>
+		<dd>
+			This option allows you to customize the behavior of the transformation command in respect to if a run is interpreted as successful or not.
+			The script you can specify here is a Groovy script that is evaluated against the transformation statistics.
+			You can use the <code>-statisticsOut</code> parameter to get an idea of the contents of the statistics.<br>
+			A success evaluation script could for example look like this:<br>
+			<pre><code>// the transformation must have been completed
+assert aggregated['eu.esdihumboldt.hale.transform'].report.completed == true
```

Comment:
In case we don't want to make the charset for the eval script configurable, we should add here that the script is expected to be in UTF-8.

Reply:
Added a note to the documentation.